### PR TITLE
[3.4] Fixed many spelling issues found in the docs

### PR DESCRIPTION
**Pull request by wouterj**
I was planning to do this for a long time: Run the [Sphinx spelling checker](https://pypi.org/project/sphinxcontrib.spelling/) on the Symfony documentation.

This PR fixes quite a lot of spelling issues found in the files starting with `b` and `c` for the 3.4 branch. I think it's good to have these fixed, so I'm happy to merge this myself and fix any merge conflicts that might occur while doing the up-merge once.

Meanwhile, I'm also preparing a "word list" of words that the spelling checker doesn't recognize. The spelling checker uses Enhant, which is [also available for PHP](https://www.php.net/manual/en/book.enchant.php). So we might be able to add this to DOCtor when all issues are fixed.

cc @symfony/team-symfony-docs

### CODE REVIEW

File: components/translation/usage.rst [hunk 1]
```diff
@@ -411,17 +411,19 @@ use for translation::
...
 
-Retrieving the Message Catalogue
---------------------------------
+.. _retrieving-the-message-catalogue:
 
-In case you want to use the same translation catalogue outside your application
+Retrieving the Message Catalog
+------------------------------
+
+In case you want to use the same translation catalog outside your application
```
Comment (OskarStark) [inline components/translation/usage.rst]: outside of ?

File: configuration/multiple_kernels.rst [hunk 2]
```diff
@@ -148,10 +148,10 @@ The ``bin/console`` script used to run Symfony commands always uses the default
 to execute console commands using the new kernel, duplicate the ``bin/console``
 script and rename it (e.g. ``bin/api``).
 
-Then, replace the ``AppKernel`` instantiation by your own kernel instantiation
-(e.g. ``ApiKernel``) and now you can execute commands using the new kernel
-(e.g. ``php bin/api cache:clear``) Now you can use execute commands using the
-new kernel.
+Then, replace the ``AppKernel`` instant by your own kernel instant (e.g.
```
Comment (OskarStark) [inline configuration/multiple_kernels.rst]: instance?

File: configuration/multiple_kernels.rst [hunk 3]
```diff
@@ -148,10 +148,10 @@ The ``bin/console`` script used to run Symfony commands always uses the default
...
+``ApiKernel``) and now you can execute commands using the new kernel (e.g.
+``php bin/api cache:clear``) Now you can use execute commands using the new
```
Comment (OskarStark) [inline configuration/multiple_kernels.rst]: and now you can...

and again 

Now you can....

BTW there is a dot missing before „Now“ as it starts a new sentence

Comment (OskarStark) [reply configuration/multiple_kernels.rst]: Or the last sentence should be removed, looks like a duplicate

File: contributing/code/conventions.rst [hunk 4]
```diff
@@ -72,10 +72,10 @@ must be used instead (where ``XXX`` is the name of the related thing):
 
 .. note::
 
-    While "setXXX" and "replaceXXX" are very similar, there is one notable
-    difference: "setXXX" may replace, or add new elements to the relation.
-    "replaceXXX", on the other hand, cannot add new elements. If an unrecognized
-    key is passed to "replaceXXX" it must throw an exception.
+    While ``setXXX`` and ``replaceXXX`` are very similar, there is one notable
```
Comment (OskarStark) [inline contributing/code/conventions.rst]: What about adding () after XXX to make clear it’s a method name?

File: contributing/community/review-comments.rst [hunk 5]
```diff
@@ -74,12 +74,12 @@ Assume everyone is intelligent and well-meaning.
...
 .. tip::
 
-    Good questions avoid judgement and avoid assumptions about the author's perspective.
+    Good questions avoid judgment and avoid assumptions about the author's perspective.
 
     Maybe you can ask for clarification? Suggest an alternative?
     Or provide a simple explanation *why* you disagree with their proposal.
 
-    * ``This looks wrong. Are you sure it's correct?`` (eg. typo/syntax error)
+    * ``This looks wrong. Are you sure it's correct?`` (e.g.. typo/syntax error)
```
Comment (OskarStark) [inline contributing/community/review-comments.rst]: 2 dots

**Review by OskarStark — CHANGES_REQUESTED** [review]
Amazing work Wouter 🥰

File: components/property_access.rst [hunk 6]
```diff
@@ -112,9 +112,9 @@ Using Getters
 ~~~~~~~~~~~~~
 
 The ``getValue()`` method also supports reading using getters. The method will
-be created using common naming conventions for getters. It camelizes the
-property name (``first_name`` becomes ``FirstName``) and prefixes it with
-``get``. So the actual method becomes ``getFirstName()``::
+be created using common naming conventions for getters. It transform the
+property name to camelCase (``first_name`` becomes ``FirstName``) and prefixes
```
[the remark this answers is not in the document]
Comment (wouterj) [reply components/property_access.rst]: > Studly caps is a form of text notation in which the capitalization of letters varies by some pattern, or arbitrarily, usually also omitting spaces between words and often omitting some letters, for example, StUdLyCaPs or STuDLyCaPS.
> -- https://en.wikipedia.org/wiki/Studly_caps



> Camel case is often used for variable names in computer programming. Some programming styles prefer camel case with the first letter capitalised, others not. For clarity, this article calls the two alternatives upper camel case (initial uppercase letter, also known as Pascal case) and lower camel case (initial lowercase letter, also known as Dromedary case).
> -- https://en.wikipedia.org/wiki/Camel_case

So I would say either PascalCase or CamelCase fit best here and I think camel case is more commonly known among developers

File: components/translation/usage.rst (hunk 1 above)
Comment (wouterj) [reply components/translation/usage.rst]: https://english.stackexchange.com/questions/9700/outside-or-outside-of

So this is a real hard one. Seems like "outside of" is more American English, but it also has the meaning "except for" and not "outside". So I would say "outside" (without "of") is the right word here.

**Review by dbrumann — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Pull request merged**